clientdb/batch_snapshot.go

```diff
@@ -382,8 +383,19 @@ func getSnapshotBuckets(tx *bbolt.Tx) (*bbolt.Bucket, *bbolt.Bucket,
 }
 
 func serializeLocalBatchSnapshot(w io.Writer, b *LocalBatchSnapshot) error {
+	// The previous batch versions had a single clearing price. We'll
+	// extract it from the map which will only contain one entry for that
+	// older version. For new batch versions we just store a zero price.
+	var clearingPrice order.FixedRatePremium
+	if len(b.ClearingPrices) == 1 {
```

Comment:
keep in mind that this might have only one element for new version as well, if only one bucket had matches.

Reply:
Removed, based on Wilmer's suggestion above.